hypothesis-python/tests/cover/test_control.py

```diff
@@ -19,10 +19,13 @@
 
 import pytest
 
+from hypothesis import Verbosity, given, settings, reporting
```

Comment:
Split out `from hypothesis.reporting import with_reporter` to a separate line (and sort appropriately).